### ISSUE TITLE
Prepare for HL7 FHIR Connectathon

### ISSUE THREAD

**Issue by brick-green** [CDCgov/prime-reportstream#9120]
## Problem statement

ReportStream engineers will be attending the FHIR Connectathon. The track includes intake of an at-home COVID test in FHIR format then delivery to a downstream system. The intake and delivery of a sample message is useful to determine current readiness of ReportStream for this effort. 

## What you need to know

[Example FHIR message](http://build.fhir.org/ig/HL7/home-lab-report/Bundle-bundle-at-home-in-vitro-test-results-p-noelle.html). You can find more [here](http://build.fhir.org/ig/HL7/home-lab-report/implementation_guide.html).

We will use APIKey auth instead of the smart-on-fhir server-server auth since that is easier for folks to deal with.

UP currently only supports sending in HL7 (See FHIRTranslator.doWork). Inside that function, instead of converting bundle to HL7, we want to do something like this, probably: `FhirContext.forR4().newJsonParser().encodeResourceToString(updatedBundle)`

PLAN:
1. (REQUIRED) Complete https://github.com/CDCgov/prime-reportstream/issues/9144 to allow sending FHIR. This is currently assigned to Jimmy from the platform team as part of ETOR work.
2. (NICE TO HAVE) Create special branch once 9144 is complete to insert Provenance resource if message does not have it. RS will throw an error during route step if Provenance does not exist since it uses it for routing. If we don't do this, we will have to manually edit messages before sending them to RS.
3. Create Demo environment for the special branch to be used by connect-a-thon. Questions: Will this work? Firewalls?
4. Test sample message. Is it unchanged as it moves through the pipeline?
6. Look at the test scenarios. Have a sample postman call with ApiKey senders can share.

[At-Home In-Vitro Info Site](https://confluence.hl7.org/pages/viewpage.action?pageId=161055948)

## Acceptance criteria

- [ ] ...

## To do

- [ ] ...

**Comment by LaFreda1** [comment]
@brick-green can we close this?
